metricbeat/docs/developer-guide/creating-beat-from-metricbeat.asciidoc

```diff
@@ -27,7 +27,17 @@ The first step is to get the metricbeat source code:
 go get github.com/elastic/beats/metricbeat
 ----
 
-This will clone the beats repository into `GOPATH`. Now change to the directory where you want to create the beat.
+This will clone the beats repository into `GOPATH`. By default `go get`  fetches the master branch. To build your beat
+on a specific version of libbeat, check out the specific branch ({doc-branch} in the example below):
+
+[source,bash,,subs="attributes"]
+----
+cd $GOPATH/src/github.com/elastic/beats
+git checkout {doc-branch}
```

Comment:
@dedemorton I tried `sub="attributes"` here so the variable gets replaced. This does not seem to work. If I leave out one , I get a different error. Any idea on how this is best done?